src/EnergyPlus/LowTempRadiantSystem.cc

```diff
@@ -1615,6 +1621,22 @@ namespace LowTempRadiantSystem {
         }
     }
 
+    LowTempRadiantSetpointTypes RadiantSystemBaseData::processRadiantSystemSetpointInput(std::string const& controlInput,
+                                                                                         std::string const& controlInputField)
+    {
+        if (UtilityRoutines::SameString(controlInput, "HalfFlowPower")) {
+            return LowTempRadiantSetpointTypes::halfFlowPower;
+        } else if (UtilityRoutines::SameString(controlInput, "ZeroFlowPower")) {
+            return LowTempRadiantSetpointTypes::zeroFlowPower;
+        } else {
```

Comment:
Can't actually get here based on how you have written the IDD changes, but it's fine.